### ISSUE TITLE
How to calculate TP , TN, FP and FN ?

### ISSUE THREAD

**Issue by vishnuvardhan58** [soumyaiitkgp/Custom_MaskRCNN#3]
@soumyaiitkgp  I am using my own dataset which consists of 1235 images in total. I have divided 864 images into a training set and 371 images in the test set. After the training process is done, I am trying to make predictions on each image in my test set. I know that there is a "compute_ap" function which returns the mAp value after sending **"gt_bbox, gt_class_id, gt_mask, r["rois"], r["class_ids"], r["scores"], r['masks']**" as parameters. But I want to know the values of TP, TN, FP, and FN of the entire test set? Can you please help me in finding these values? Is there any predefined function available in the model which returns these values?

**Comment by soumyaiitkgp** [comment]
Thank you @vishnuvardhan58  for raising the issue. If someone is looking for a possible solution, follow the discussion for the above query visit here: 
https://github.com/matterport/Mask_RCNN/issues/2408
